Modules/_elementtree.c

```diff
@@ -675,11 +675,11 @@ _elementtree_Element_clear_impl(ElementObject *self)
     dealloc_extra(self);
 
     Py_INCREF(Py_None);
-    Py_DECREF(JOIN_OBJ(self->text));
+    _clear_joined_ptr(&self->text);
```

Comment:
Setting `self->text` to NULL is not safe. In this case the "text" getter will return NULL without setting an exception.